modules/client/src/addresslistVoting/internal/client/methods.ts

```diff
@@ -307,16 +307,18 @@ export class AddresslistVotingClientMethods extends ClientCore
     }
 
     try {
-      await this.graphql.ensureOnline();
+      this.graphql.assertClient();
       const client = this.graphql.getClient();
```

Comment:
Would there be a downside if we moved `assertClient()` into `getClient()` ?

Reply:
yes no problem with that 👍